src/librekey/key_store_g10.cpp

```diff
@@ -1105,7 +1105,7 @@ rnp_key_store_g10_from_src(rnp_key_store_t *         key_store,
 {
     const pgp_key_t *pubkey = NULL;
     pgp_key_t        key = {0};
-    pgp_key_pkt_t    seckey = {0};
+    pgp_key_pkt_t    seckey = {};
```

Comment:
Are we moving from `{0}` towards `{}` syntax?

Reply:
@rrrooommmaaa this came out after moving from C99 to C++11. Within C code = {0} will fill struct with zeroes, while C++ would assign 0 to first parameter (and in this particular case - throw an error since first parameter is of pgp_pkt_type_t which cannot be assigned from int by default). For most struct's it still works since first parameter is of int or similar type.